specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json

```
@@ -4419,6 +4661,122 @@
         }
       }
     },
+    "HubRouteTableProperties": {
+      "properties": {
+        "routes": {
+          "type": "array",
+          "description": "List of all routes.",
+          "items": {
+            "$ref": "#/definitions/HubRoute"
+          }
+        },
+        "labels": {
+          "type": "array",
+          "description": "List of labels associated with this route table.",
+          "items": {
+            "type": "string"
+          }
+        },
+        "associatedConnections": {
+          "type": "array",
+          "description": "List of all connections associated with this route table.",
+          "readOnly": true,
+          "items": {
+            "$ref": "./network.json#/definitions/SubResource"
+          }
+        },
+        "propagatingConnections": {
+          "type": "array",
+          "description": "List of all connections that advertise to this route table.",
+          "readOnly": true,
+          "items": {
+            "$ref": "./network.json#/definitions/SubResource"
+          }
+        },
+        "provisioningState": {
+          "$ref": "./network.json#/definitions/ProvisioningState",
+          "readOnly": true,
+          "description": "The provisioning state of the RouteTable resource."
+        }
+      },
+      "description": "Parameters for RouteTable."
+    },
+    "HubRouteTable": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/HubRouteTableProperties",
+          "description": "Properties of the RouteTable resource."
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the resource that is unique within a resource group. This name can be used to access the resource."
+        },
+        "etag": {
+          "type": "string",
+          "readOnly": true,
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        },
+        "type": "Microsoft.Network/virtualHubs/hubRouteTables"
```

@pilor : can you check this : added the type property (want to verify it)

---

It should be a string property similar to:

```suggestion
        "type": {
          "readOnly": true,
          "type": "string",
          "description": "Resource type."
        },
```

---

done thank you